libs/ml/include/ml/layers/convolution_1d.hpp

```diff
@@ -18,17 +18,9 @@
 //------------------------------------------------------------------------------
 
 #include "ml/core/subgraph.hpp"
```

Comment:
move includes into source across all layers

Reply:
but I need these include to resolve namespaces:

  using SizeType      = fetch::math::SizeType;
  using WeightsInit   = fetch::ml::ops::WeightsInitialisation;

Reply:
I tried to forward declare subgraph, WeightsInitialisation, LayerConvolution1DSaveableParams, Activation etc,  but they require definition in header:

Convolution1D(SizeType const output_channels, SizeType const input_channels,
                SizeType const kernel_size, SizeType const stride_size,
                details::ActivationType const activation_type = details::ActivationType::NOTHING,
                std::string const &           name            = "Conv1D",
                WeightsInit const             init_mode       = WeightsInit::XAVIER_GLOROT,
                SizeType const                seed            = 123456789);